pool/app/pool_tenant/event.ml

```diff
@@ -82,5 +82,5 @@ let handle_event pool : event -> unit Lwt.t = function
     Lwt.return_unit
   | GtxApiKeyUpdated (tenant, gtx_api_key) ->
```

Comment:
Is there an event for deleting the key? How to remove it or is it possible to set the key to an empty value?

Reply:
No currently there is no way to delete it. I added a note to the issue. Not sure if I will be able to do that today

Reply:
Done